cmake/DefaultTargetProperties.cmake

```diff
@@ -0,0 +1,206 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+function(set_default_target_properties target_name)
+    #-------------------------------------------------------------------
+    # C++ standard
+
+    # Minimum C++ standard
+    target_compile_features(${target_name} PRIVATE cxx_std_14)
```

Comment:
As above, for VFX22, C++ standard targeted is C++ 17.
See https://vfxplatform.com/

Reply:
Switched to C++17 in ca741452